_data/simple-icons.json

```
@@ -4129,6 +4129,11 @@
             "hex": "FF3008",
             "source": "https://www.doordash.com/about/"
         },
+        {
+            "title": "Dota 2",
+            "hex": "BF2E1A",
+            "source": "https://en.m.wikipedia.org/wiki/File:Dota_logo.svg"
```

We can use the Wikimedia version which is the source of the Wikipedia one:
```suggestion
            "source": "https://commons.wikimedia.org/wiki/File:Dota_logo.svg"
```